contributor/es/04-mechanics-of-contributing-article-es.asciidoc

```diff
@@ -0,0 +1,158 @@
+== Mecánica de la contribución
+
+¿Estás preparado para empezar a contribuir a los proyectos/repositorios de otros equipos?
+¿Quieres reducir tus bloqueos colaborando en vez de gestionar?
+En esta sección se ofrecen consejos prácticos y se destacan los puntos que hay que recordar al realizar una contribución en InnerSource. Permite que el equipo anfitrión y tú tengais una experiencia lo más agradable posible, sentando las bases para más contribuciones y una gran colaboración.
+
+Este artículo se divide en los tres pasos que probablemente experimentarás
+
+* <<preparación-para-el-trabajo, solicitar tu oportunidad de contribución y prepararte para trabajar en ella>>
+* <<crear-el-pull-request, crear realmente la contribución>>
+* <<enviar-el-pull-request, pulir y envolver bien el regalo y presentarlo al equipo anfitrión>>
+
+Si tu contribución es más grande, posiblemente pasarás por (algunos) de estos pasos repetidamente mientras iteras hacia tu objetivo común.
+Es muy probable que, a medida que lo hagas, todo te resulte cada vez más natural; incluso puede que te preguntes por qué hacías otra cosa antes.
+
+=== Preparación para el trabajo
+
+==== Tiempos de entrega
+
+Una diferencia clave es el tiempo de entrega.
+Con cada contribución por primera vez, usted llega a un nuevo equipo (anfitrión).
+En consecuencia, tendrás que conocer su base de código, las tecnologías utilizadas y también su entorno de desarrollo preferido (piensa en el marco de pruebas, el sistema de compilación).
+Incluso en los casos en los que este tipo de herramientas están estandarizadas, cada equipo habrá desarrollado algunas peculiaridades individuales.
+Además de la parte técnica, puedes encontrarte con diferencias en la comunicación (piensa en las revisiones de código).
+Incluso si vuelves después de un tiempo, las formas y los miembros de los equipos pueden haber cambiado.
+Tómate tu tiempo como lo harías para ponerte al día con un amigo al que hace tiempo que no ves y al que ahora visitas.
+
+Date el tiempo suficiente.
+Empieza con la suficiente antelación para que tu trabajo esté disponible para que lo aproveches en el momento que lo necesites.
+Es mejor añadir más tiempo de holgura al principio: ya te harás una idea de los plazos de entrega una vez que trabajes con el equipo anfitrión.
+A menudo, notará una reducción en el tiempo de entrega por equipo anfitrión después de hacer algunas contribuciones exitosas a ese equipo anfitrión.
+Este efecto puede observarse también en el código abierto, puede leer más sobre ello <<construcción de confianza a través de la colaboración, aquí>>.
```

Comment:
This is a link to a resource down below and it needs to have the same name to link properly
```suggestion
Este efecto puede observarse también en el código abierto, puede leer más sobre ello <<Creación de confianza a través de la colaboración, aquí>>.
```